Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -165,47 +166,47 @@ public void TestInterestEarnedIsDisplayed()
         public void TestEndDateDataIsCorrect()
 
         {
-            browser.FindElement(By.Id("term")).SendKeys("7");
+            browser.FindElement(By.XPath("//input [@id = 'term']")).SendKeys("7");
             DateTime expected = DateTime.Today.AddDays(+7);
            
-            string EndDate = browser.FindElement(By.Id("endDate")).GetAttribute("value");
+            string EndDate = browser.FindElement(By.XPath("//input [@id = 'endDate']")).GetAttribute("value");
 
             Assert.AreEqual(expected.ToString("dd/MM/yyyy"), EndDate);
         }
 
         [Test]
         public void TestInterestEarnedLayout()
         {          
-            string actual = browser.FindElement(By.XPath("/html/body/div/div/table/tbody/tr[7]/th[1]")).GetAttribute("align");
+            string actual = browser.FindElement(By.XPath("//tr[7]//th [contains(text ), 'Interest earned')]")).GetAttribute("align");
```

Comment:
Not clear what does it mean: //tr[7].
```suggestion
            string actual = browser.FindElement(By.XPath("//th [contains(text(), 'Interest earned')]")).GetAttribute("align");
```

Reply:
agree, thanks. I had a syntaxis problem here